### ISSUE TITLE
Double slash in asset files' url

### ISSUE THREAD

**Issue by yiichou** [zhe/hugo-theme-slim#5]
In your code, you load the assets like these:

```
<link rel="stylesheet" href="{{ .Site.BaseURL }}/css/slim.css">
<link rel="stylesheet" href="{{ .Site.BaseURL }}/css/highlight.min.css">
```

But, `Site.BaseURL` has been often end with a slash already.
It lead to get the js/css url like:

```
http://localhost//css/highlight.min.css
```

Double slash urls are still valid, but the browser need an extra 301 redirect to got the right file.

These code should be:

```
<link rel="stylesheet" href="{{ .Site.BaseURL }}css/slim.css">
<link rel="stylesheet" href="{{ .Site.BaseURL }}css/highlight.min.css">
```

**Comment by zhe** [comment]
Hey Ivan, you're right. I'll fix this. 👍

**Comment by zhe** [comment]
Fixed by commit fc4110b564fabc44020eaa128c3de604e6b7b403 
Thanks for pointing it out.

Cheers,
Zhe
